src/opm/output/eclipse/Summary.cpp

```diff
@@ -688,6 +742,70 @@ inline quantity potential_rate( const fn_args& args ) {
     return { sum, rate_unit< phase >() };
 }
 
+template < bool isGroup = true, bool Producer = true, bool waterInjector = false, bool gasInjector = false>
```

Comment:
Is the default initialization used/needed?

Reply:
I agree, I do not think the default is needed - I will remove it.